src/main/cpp/Robot.cpp

```diff
@@ -35,6 +38,16 @@ Robot::Robot() {
         "Right Side Shoot Three Balls",
         std::bind(&Robot::AutoRightSideShootThreeInit, this),
         std::bind(&Robot::AutoRightSideShootThreePeriodic, this));
+
+    nt::NetworkTableInstance inst = nt::NetworkTableInstance::GetDefault();
+    std::shared_ptr<nt::NetworkTable> table = inst.GetTable("Diagnostics");
+    m_flywheelEntry = table->GetEntry("flywheel");
+    m_drivetrainLeftEntry = table->GetEntry("drivetrain-Left");
+    m_drivetrainRightEntry = table->GetEntry("drivetrain-Right");
+    m_drivetrainGyroEntry = table->GetEntry("drivetrain-Gyro");
+    m_turretEntry = table->GetEntry("turret");
+    m_upperConveyorEntry = table->GetEntry("upper-Conveyor");
+    m_lowerConveyorEntry = table->GetEntry("lower-Conveyor");
```

Comment:
Make the entry name the key you want to see in OutlineViewer. `drivetrain-Left` is weird. Maybe `Drivetrain left encoder`? The length of the name won't matter because we never reference it from robot code.